ci/scripts/go_bench_adapt.py

```diff
@@ -79,7 +79,11 @@ def _transform_results(self) -> List[BenchmarkResult]:
                         "params": '/'.join(pieces[1:]),
                     },
                     run_reason=RUN_REASON,
-                    github=None, # scrape info from cloned repo
+                    github={
+                        "repository": os.environ["GITHUB_REPOSITORY"],
+                        "commit": os.environ["GITHUB_SHA"],
+                        "pr_number": None,  # we currently only run this on the default branch
```

Comment:
If this ever changes, it will be important to scrape the PR number from a GHA environment variable. [This issue](https://github.com/actions/checkout/issues/58) has many suggestions on how to do that.

Reply:
yea, it's something easy enough to grab from an Env var or to have passed in via the GHA itself.